test/operations/operators/createOperatorClass.spec.ts

```diff
@@ -30,21 +108,243 @@ describe('operations', () => {
 
         expect(statement).toBeTypeOf('string');
         expect(statement).toStrictEqual(
-          `CREATE OPERATOR CLASS "gist__int_ops" DEFAULT FOR TYPE "_int4" USING "gist"  AS
-  FUNCTION 3 "&&"();`
+          `CREATE OPERATOR CLASS "gist__int_ops" DEFAULT FOR TYPE "_int4" USING "gist" AS
+  OPERATOR 3 "&&",
+  OPERATOR 6 "="(anyarray, anyarray),
+  OPERATOR 7 "@>",
+  OPERATOR 8 "<@",
+  OPERATOR 20 "@@"(_int4, query_int),
+  FUNCTION 1 "g_int_consistent"(internal, _int4, smallint, oid, internal),
+  FUNCTION 2 "g_int_union"(internal, internal),
+  FUNCTION 3 "g_int_compress"(internal),
+  FUNCTION 4 "g_int_decompress"(internal),
+  FUNCTION 5 "g_int_penalty"(internal, internal, internal),
+  FUNCTION 6 "g_int_picksplit"(internal, internal),
+  FUNCTION 7 "g_int_same"(_int4, _int4, internal);`
```

Comment:
If I throw that against a postgres, I get:

````ts
ERROR:  syntax error at or near ","
LINE 2:   OPERATOR 3 "&&",
````

It get's slightly better, when you remove the quotes:

As shown by the example from [postgres](https://www.postgresql.org/docs/current/sql-createopclass.html):

````ts
CREATE OPERATOR CLASS gist__int_ops
    DEFAULT FOR TYPE _int4 USING gist AS
        OPERATOR        3       &&,
        OPERATOR        6       = (anyarray, anyarray),
        OPERATOR        7       @>,
        OPERATOR        8       <@,
        OPERATOR        20      @@ (_int4, query_int),
        FUNCTION        1       g_int_consistent (internal, _int4, smallint, oid, internal),
        FUNCTION        2       g_int_union (internal, internal),
        FUNCTION        3       g_int_compress (internal),
        FUNCTION        4       g_int_decompress (internal),
        FUNCTION        5       g_int_penalty (internal, internal, internal),
        FUNCTION        6       g_int_picksplit (internal, internal),
        FUNCTION        7       g_int_same (_int4, _int4, internal);
````

Sadly even with that I get an error:

````txt
ERROR:  operator does not exist: integer[] && integer[]
````

But that might be caused by my test setup being cobbled together.
(or me not having a clue, what this does/it is used for)